src/app/qgisapp.cpp

```diff
@@ -978,6 +1026,16 @@ QgisApp::QgisApp( QSplashScreen *splash, bool restorePlugins, bool skipVersionCh
 #ifdef ANDROID
   toggleFullScreen();
 #endif
+  QgsDebugMsg("PROFILE TIMES");
```

Comment:
I'd move this to the profiler class. Then add a nice ascii-art style bar chart to make it obvious where the bottlenecks are, eg:

```
PROFILE TIMES TOTAL - 12.811 
00o........ - SETTING UP UI - 0.208
o.......... - CHECKING DATABASE - 0.037
0000oo..... - CREATING THEME FOLDER - 0.003
00000000000 - some slow thing - 1111.01
etc
```